frontend/packages/vkt/public/i18n/fi-FI/privacy.json

```diff
@@ -0,0 +1,119 @@
+        "description2": "Henkilötietoja voidaan luovuttaa tieteellistä tai historiallista tutkimusta taikka tilastointia varten tietosuojalain (1050/2018) 4 §:n mukaisin edellytyksin.",
+        "description3": "Tietojärjestelmän palveluntarjoajat (henkilötietojen käsittelijät) pääsevät tarkastelemaan rekisterin sisältämiä henkilötietoja Opetushallituksen määrittämässä laajuudessa.",
+        "description4": "Tietoja ei luovuteta suoramarkkinointiin.",
+        "heading": "Henkilötietojen vastaanottajat tai vastaanottajaryhmät"
+      },
+      "registeredRights": {
+        "description1": "Rekisteröidyllä on oikeus saada rekisterinpitäjältä vahvistus siitä, että häntä koskevia henkilötietoja käsitellään tai että niitä ei käsitellä. Rekisterinpitäjän on toimitettava pyynnöstä jäljennös käsiteltävistä henkilötiedoista.",
+        "description2": "Rekisteröidyllä on oikeus vaatia, että rekisterinpitäjä oikaisee ilman aiheetonta viivytystä rekisteröityä koskevat epätarkat ja virheelliset henkilötiedot. Rekisteröidyn tulee yksilöidä ja perustella, mitä tietoa hän vaatii korjattavaksi, mikä on hänen mielestään oikea tieto ja miten korjaus halutaan tehtäväksi.",
+        "description3": "Mikäli käsittely perustuu tietosuoja-asetuksen 6 artiklan 1a -kohtaan tai 9 artiklan 2a -kohtaan eli rekisteröidyn suostumukseen, on rekisteröidyllä oikeus tietojen poistamiseen.",
+        "description4": "Rekisteröidyllä on oikeus käsittelyn rajoittamiseen tietyissä tilanteissa.",
+        "description5": "Rekisteröidyllä on oikeus siihen, että rekisterinpitäjä ilmoittaa henkilötietojesi oikaisusta tai poistosta tai käsittelyn rajoituksesta sille, jolle tietoja on edelleen luovutettu, mikäli tietoja luovutetaan eteenpäin.",
+        "description6": "Rekisteröidyllä on oikeus saada tiedot siirrettyä järjestelmästä toiseen, jos käsittely suoritetaan automaattisesti.",
+        "description7": "Oikeuksien käyttöön liittyvät pyynnöt tulee osoittaa rekisterin yhteyshenkilölle: Opetushallitus, PL 380, 00531 Helsinki. Tarkastuspyyntöön rekisteröidyn tulee liittää tietojen etsimiseen tarvittavat tiedot (nimi ja henkilötunnus).",
+        "description8": "Jos rekisteröidyn käyttämästä henkilötietojen tarkastusoikeudesta on kulunut vähemmän kuin yksi vuosi, voi rekisterinpitäjä periä tietojen antamisesta aiheutuvat hallinnollisiin kustannuksiin perustuvan maksun (artikla 12 [5]).",
+        "heading": "Rekisteröidyn oikeudet",
+        "rights": {
+          "right1": "Oikeus saada pääsy henkilötietoihin",
+          "right2": "Oikeus tietojen oikaisemiseen",
+          "right3": "Oikeus tietojen poistamiseen",
+          "right4": "Oikeus käsittelyn rajoittamiseen",
+          "right5": "Vastustamisoikeus",
+          "right6": "Oikeus siirtää tiedot järjestelmästä toiseen"
+        }
+      },
+      "registerName": {
+        "description": "Valtionhallinnon kielitutkintojen erinomaisen taidon tutkinnon ilmoittautuminen ja tutkintomaksun maksaminen",
```

Comment:
Tänne vois laittaa pisteen. Tais olla, että ruudunlukija lukee sen silloin paremmin tjsp.

Reply:
Tässä selosteessa on oikeastaan aika paljon tälläisia kohtia, jotka eivät ole oikeastaan lauseita, mutta ruudunlukijaa varten pisteet varmaan helpottaisi tekstin jäsentelyä / lukemista. Itse en ehkä ainakaan suoralta kädeltä heittäis pisteitä näihin, tääkin tässä rekisterin nimi. Noi alemmat kaks taas lakiperusteita. Niissä vois toki sellaista miettiä josko niissä olis se lakiin viittaava linkki lopussa ja sitä ennen kaksoispistettä tms. jos helpottais luettavuutta.

Reply:
Emmaa vois toki pyytää myös ottaa kantaa tähän että koittaisko saavutettavuutta parantaa jotenkin jäsentelemällä tekstiä uusiksi?

Reply:
Voisin tehdä tähän ehkä mergen tai rebasen ja viedä untuvalle niin Emma voi tsekkaa. Tuota ennen voi sinänsä halutessa myös mergailla noita muita pullareita, latausindikaattorit + api docs.